sw/device/lib/dif/dif_hmac.h

```diff
@@ -0,0 +1,364 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 1.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_LIB_DIF_DIF_HMAC_H_
+#define OPENTITAN_SW_DEVICE_LIB_DIF_DIF_HMAC_H_
+
+#include "sw/device/lib/base/mmio.h"
+#include <stddef.h>
+#include <stdint.h>
+
+/**
+ * HMAC interrupt configuration.
+ *
+ * Enumeration used to enable, disable, test and query the UART interrupts.
+ * Please see the comportability specification for more information:
+ * https://docs.opentitan.org/doc/rm/comportability_specification/
+ */
+typedef enum dif_hmac_interrupt {
+  //! HMAC is done.
```

Comment:
Can you file a follow-up issue to go through and replace your `//!` and `/*!` comments with `/**` comments? Let's not block this PR, though.